lib/rmg/TreasurePlacer.cpp

```diff
@@ -651,13 +650,10 @@ ObjectInfo * TreasurePlacer::getRandomObject(ui32 desiredValue, ui32 currentValu
 	else
 	{
 		int r = generator.rand.nextInt(1, total);
-		
+		auto sfun = [](const std::pair<ui32, ObjectInfo *> & rhs, const ui32 lhs) -> bool { return static_cast<int>(rhs.first) < lhs; };
```

Comment:
Formatting - please no one-liner lambda's
Also - what `sfun` means? `sorter`, `sorterFunctor` or simply `functor` is probably better

Reply:
sfun is sorterfunction. I can rename it to sorter, if you wish.

Reply:
Yeah, please minimize abbreviating things

Reply:
renamed to sorter and uninlined.